docs/index.txt

```diff
@@ -117,10 +117,19 @@ AVATAR_GRAVATAR_BACKUP
     ``Avatar`` instance is found in the system for the given user. Defaults to
     True.
 
+GRAVATAR_DEFAULT
```

Comment:
Well that's embarrassing. Thanks for pointing it out!

Fixed in https://github.com/davegaeddert/django-avatar/commit/200e4ae047e883451af6a1fc55bbc931fcfaddfa